controller/migrations/data/0001.sh

```diff
@@ -1,19 +1,15 @@
 #!/usr/bin/env bash
 
-ETCD_PORT=${ETCD_PORT:-4001}
-ETCD="$HOST:$ETCD_PORT"
```

Comment:
@bacongobbler If I am not mistaken, at the time this migration script runs, the controller's boot script has already created the `ETCDCTL_PEERS` environment variable if it didn't already exist.

As proof, all integration tests are passing locally and in CI on an "old style" cluster, straight-out-of-the-box and not modified in any way.  In other words, I have not set `ETCDCTL_PEERS` explicitly on any host in the test cluster, so an empty value for `ETCDCTL_PEERS` gets passed by each system unit into its corresponding container and the modified logic in each boot script is constructing `ETCDCTL_PEERS` successfully since it hasn't been set.

By contrast, I _do_ set `ETCDCTL_PEERS` on each host that does not run etcd locally in my live clusters.  And things work great there too.

I'm pretty confident this achieves what I set out to do without breaking anything existing... but given the magnitude of the change, continued scrutiny is welcome and appreciated.